specs/system_config.md

```diff
@@ -13,11 +13,15 @@
   - [`unsafeBlockSigner` (`address`)](#unsafeblocksigner-address)
 - [Writing the system config](#writing-the-system-config)
 - [Reading the system config](#reading-the-system-config)
+- [Ecotone Activation](#ecotone-activation)
+  - [Constants](#constants)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 The `SystemConfig` is a contract on L1 that can emit rollup configuration changes as log events.
 The rollup [block derivation process](./derivation.md) picks up on these log events and applies the changes.
+The invariant must be held that the L2 view of the `SystemConfig` MUST always be consistent
```

Comment:
What exactly does the "L2 view" mean here?

Reply:
I assume it means the values set in the L1 attributes tx?

Reply:
The `op-node` has a view of the `SystemConfig`. The view is set thru the L1 attributes transaction